packages/teams-js/src/public/app.ts

```diff
@@ -2,18 +2,23 @@
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
+/**
+ * v2 APIs telemetry file: All of APIs in this capability file should send out API version v2 ONLY
```

Comment:
It would help enforce this if you made a const for the version number to use in this file and then references that const everyone you say V_2 right now.

Reply:
Pretend I typed this for every file with a comment like this

Reply:
```
const appTelemetryVersion: ApiVersionNumber = ApiVersionNumber.V_2;
```
Then you would use `appTelemetryVersion` instead of `ApiVersionNumber.V_2` below. That would make it a lot harder for other people in the future to add the "wrong" version. If they go look at another function in the same file they will see that they are using `appTelemetryVersion` and use that one too.